packages/earth-map/src/components/filter-by/FilterBy.tsx

```diff
@@ -110,7 +113,7 @@ const FilterBy = (props: IProps) => {
           <Typography component="span" variant="button">
             {t('Filters')}
           </Typography>{' '}
-          {numberOfFilters > 0 && (
+          <Fade in={numberOfFilters > 0}>
```

Comment:
I think the effect is not perceptible to the user and only adds more load on interaction. It also creates a weird effect where you can see "CLEAR (0)" before fading out completely.

I suggest we stick to the original interaction.